packages/edit-site/src/components/editor-canvas-container/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * WordPress dependencies
+ */
+import { Children, cloneElement, useState, useMemo } from '@wordpress/element';
+import { createSlotFill, Button } from '@wordpress/components';
+import { ESCAPE } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { closeSmall } from '@wordpress/icons';
+import { useFocusOnMount, useFocusReturn } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../private-apis';
+import { store as editSiteStore } from '../../store';
+
+/**
+ * Returns a translated string for the title of the editor canvas container.
+ *
+ * @param {string} view Editor canvas container view.
+ *
+ * @return {string} Translated string corresponding to value of view. Default is ''.
+ */
+export function getEditorCanvasContainerTitle( view ) {
+	switch ( view ) {
+		case 'style-book':
+			return __( 'Style Book' );
+		default:
+			return '';
+	}
+}
+
+const SLOT_FILL_NAME = 'EditSiteEditorCanvasContainerSlot';
+const { Slot: EditorCanvasContainerSlot, Fill: EditorCanvasContainerFill } =
+	createSlotFill( SLOT_FILL_NAME );
+
+function EditorCanvasContainer( {
```

Comment:
Nice, I like this abstraction — should make it easier building subsequent screens (which I'm sure is why you made it like this 😄)